modules/fts/pages/fts-advance-settings-maxConcurrentPartitionMovesPerNode.adoc

```diff
@@ -0,0 +1,34 @@
+= maxConcurrentPartitionMovesPerNode 
+
+During the rebalance operations, FTS moves or builds partitions one at a time per node. This process can significantly increase the overall time taken for the rebalance operation. So, the way to speed up the rebalance operation is to enable the movement of partitions parallelly.
```

Comment:
I would drop the 2nd sentence - that this process can significantly increase the overall time taken for the rebalance operation.

I'd just mention, that we do offer a way to speed up the rebalance operation to move partitions concurrently.